README.md

```diff
@@ -10,79 +10,117 @@ REQUIREMENTS
 
 * PHP5
 * MySQL 5
-* MediaWiki 1.11+ 
+* MediaWiki 1.11+
 * phpBB 3.2
 
 INSTALL:
 =================
 
-Create a group in PHPBB for your wiki users. I named mine "Wiki". 
+Create a group in phpBB for your wiki users. I named mine "Wiki". 
 You will need to put the name you choose in the code below. 
 
-NOTE: In order for a user to be able to use the wiki they will need to 
+PHPBB GROUP NOTE: In order for a user to be able to use the wiki they will need to 
 be a member of the group you made in the step above.
 
-Put Auth_phpbb.php in /extensions/  
-Put iAuthPlugin.php in /extensions/  
-Put PasswordHash.php in /extensions/
+Put Auth_phpbb.php in /extensions/Auth_phpBB
+Put iAuthPlugin.php in /extensions/Auth_phpBB
+Put PasswordHash.php in /extensions/Auth_phpBB
 
+OPTIONAL PHPBB-TO-MEDIAWIKI USERNAME TRANSLATION FEATURE:
```

Comment:
It already has an option in the LocalSettings.php config.  It defaults to false so that default behavior is unchanged from previous versions and it's use would be deliberate.